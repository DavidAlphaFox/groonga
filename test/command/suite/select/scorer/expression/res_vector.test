plugin_register functions/vector

table_create Memos TABLE_HASH_KEY ShortText
column_create Memos categories COLUMN_VECTOR ShortText

load --table Memos
[
{"_key": "Groonga", "categories": ["full-text-search"]}
]

select \
   --filter "true" \
   --output_columns "_key, _score" \
   --scorer "(_score = _score + (vector_size(categories) > 0))" \
   --table "Memos"
