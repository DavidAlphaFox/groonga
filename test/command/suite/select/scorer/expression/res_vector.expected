plugin_register functions/vector
[[0,0.0,0.0],true]
table_create Memos TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Memos categories COLUMN_VECTOR ShortText
[[0,0.0,0.0],true]
load --table Memos
[
{"_key": "Groonga", "categories": ["full-text-search"]}
]
[[0,0.0,0.0],1]
select    --filter "true"    --output_columns "_key, _score"    --scorer "(_score = _score + (vector_size(categories) > 0))"    --table "Memos"
[[0,0.0,0.0],[[[1],[["_key","ShortText"],["_score","Int32"]],["Groonga",2]]]]
